### ISSUE TITLE
Webterm Appliance Terminal Error

### ISSUE THREAD

**Issue by gns3warrior** [GNS3/gns3-registry#278]
tl;dr 
*** VTE ***: Failed to load terminal capabilities from '/etc/termcap'
root@webterm-1:~#

I started a troubleshooting thread on the GNS3 board and was directed by AJ, one of the moderators to open an issue here. 

More background and troubleshooting that was already performed: https://gns3.com/qa/webterm-appliance-terminal-error

**Comment by ehlers** [comment]
In my environment it works well:
![webterm](https://user-images.githubusercontent.com/896406/31309126-892909ec-ab81-11e7-893b-f0626da0179f.png)

Can you please tell more about your environment? OS, GNS3 version, VNC software/version.
Please start a webterm VM, open the terminal and issue the command "echo $TERM" (as shown in my screenshot). What's the output?

**Comment by gns3warrior** [comment]
The good news is after further digging today, I was able to discern that there is actually a docker image and docker container that both need to be deleted from the GNS3 VM. 

Once I was able to delete both rather than just the docker container, it pulled a new image from the server again and the terminal behaves as intended.

Unfortunately, there is nothing I can add that will clear up why this happened in the first place. It was working, then it wasn't. For clarification, I'm using:

Windows 10 x64 Creators Update
GNS3 2.0.3
GNS3 VM 0.10.14
Tight VNC Viewer 2.7.10
